src/Factory.php

```diff
@@ -136,6 +137,7 @@ public function createLogger(): Logger
             $this->logger->pushProcessor(new UidProcessor());
             $this->logger->pushProcessor(new MemoryUsageProcessor());
             $this->logger->pushProcessor(new IdentifyProcessor());
+            $this->logger->pushProcessor(new RequestIdProcessor());
```

Comment:
suggestion to avoid building your own Uid Class and using monolog bundled implementation

````php
$processor = new UidProcessor(); 
$this->logUid = $processor->getId();
// somwhere inside Factory you may have a getter for $logUid
$this->pushProcessor($processor);
````